index.md

```diff
@@ -24,6 +24,10 @@ constitute the Trixi framework on [GitHub](https://github.com/orgs/trixi-framewo
 
   Convert output files generated with Trixi.jl to VTK
 
+* [**Libtrixi**](https://github.com/trixi-framework/libtrixi)
```

Comment:
We haven't made any "official" decision here, but so far we have consistently use it as follows:
* Write `libtrixi` when encountered in the middle of a sentence or as a standalone entity (e.g., a heading)
* Write `Libtrixi` at the beginning of sentences (to avoid the impression of a typo)
This is, as far as I can tell, similar to what other OSS libraries use.

Thus, you are right in catching this and it should be
```suggestion
* [**libtrixi**](https://github.com/trixi-framework/libtrixi)
```